packages/flutter/test/rendering/editable_gesture_test.dart

```diff
@@ -36,6 +36,8 @@ void main() {
       ),
       onSelectionChanged: (_, __, ___) { },
     );
+    editable.layout(BoxConstraints.loose(const Size(1000.0, 1000.0)));
```

Comment:
I have to update this test because now the hit test requires layout. I think this is always true in real world environment. I am not sure if i should treat this as breaking change....

Reply:
and we require layout because we look at constraints, i see